container/cmd

```diff
@@ -39,6 +50,7 @@ analyze_subdir()
         result=false
         return
     }
+    get_pure_defects
     vcs-diff-lint --print-fixed-errors "${linter_options[@]}" || result=false
```

Comment:
Thank you for the PR!

Can you please fix the PR so vcs-diff-lint is actually called just once? There are cases it runs for several minutes (pylint/mypy slowness).  Running it twice seems wasting of time.

Reply:
I agree, that calling `vcs-diff-lint` multiple times isn't the best solution.

But I think that the main problem is that `vcs-diff-lint --print-fixed-errors "${linter_options[@]}"` produces extra information that `csgrep` isn't able to parse. Do you have any ideas for improvements?

Reply:
Ah, OK - this requires another option in the tool itself.  Thanks for the clarification!